### PR TITLE
Mainnet Genesis

### PR DESCRIPTION
**Pull request by sunnya97**

### CODE REVIEW

File: app/params/genesis.go [hunk 1]
```diff
@@ -41,6 +46,98 @@ type GenesisParams struct {
...
+	genParams.DistributionParams.BaseProposerReward = sdk.MustNewDecFromStr("0.01")
+	genParams.DistributionParams.BonusProposerReward = sdk.MustNewDecFromStr("0.04")
+	genParams.DistributionParams.CommunityTax = sdk.MustNewDecFromStr("0")
+	genParams.DistributionParams.WithdrawAddrEnabled = true
+
+	genParams.GovParams = govtypes.DefaultParams()
+	genParams.GovParams.DepositParams.MaxDepositPeriod = time.Hour * 24 * 14 // 2 weeks
+	genParams.GovParams.DepositParams.MinDeposit = sdk.NewCoins(sdk.NewCoin(
+		genParams.NativeCoinMetadata.Base,
+		genParams.AirdropSupply.QuoRaw(1_000_000), // 1 millionth of airdrop supply
```
Comment (sunnya97) [inline app/params/genesis.go:104]: Change to 1000 osmo

File: app/params/genesis.go [hunk 2]
```diff
@@ -41,6 +46,98 @@ type GenesisParams struct {
...
+	))
+	genParams.GovParams.TallyParams.Quorum = sdk.MustNewDecFromStr("0.25") // 25%
```
Comment (sunnya97) [inline app/params/genesis.go:106]: Make quorum 15%

Comment (ValarDragon) [reply app/params/genesis.go:106]: I'd really prefer to keep it at 25%, it already feels low but ~somewhat justifiable due to software issues + very quick voting periods at launch

File: app/params/genesis.go (hunk 1 above)
Comment (sunnya97) [reply app/params/genesis.go:104]:
```suggestion
		genParams.AirdropSupply.QuoRaw(100_000), // 1000 OSMO
```

File: app/params/genesis.go (hunk 2 above)
Comment (sunnya97) [reply app/params/genesis.go:106]: ok

**Pull request merged**